src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Project Duke CS2103
+ * Done by Hong Jin.
+ */
+package duke;
+
+import duke.task.*;
+
+import java.util.Scanner;
+
+/**
+ * public class Ui to handle text shown to users when command is typed in to Duke.
+ */
+public class Ui {
```

Comment:
Good idea to add a UI class, definitely helps in handling functions that prints out the response